addons/web/static/tests/components/dropdown_menu_tests.js

```diff
@@ -70,6 +71,51 @@ odoo.define('web.dropdown_menu_tests', function (require) {
             dropdown.destroy();
         });
 
+        QUnit.test('only one dropdown rendering at same time (owl vs bootstrap dropdown)', async function (assert) {
+            assert.expect(6);
+            const form = await testUtils.createView({
+                View: FormView,
+                model: 'partner',
+                data: {
+                    partner: {
+                        fields: {
+                            display_name: {string: "STRING", type: 'char'}
+                        },
+                        records: [{id: 1, display_name: "partner",}]
+                    }
+                },
+                arch: `<div class="dropdown">
+                    <button class="btn dropdown-toggle" type="button" 
+                            data-toggle="dropdown" aria-expanded="false">
+                        BS Dropdown button
+                    </button>
+                    <div class="dropdown-menu">
+                        <a class="dropdown-item" href="#">BS Action</a>
+                    </div>
+                </div>`,
+            });
+            const dropdown = await createComponent(DropdownMenu, {
+                props: {
+                    items: this.items,
+                    title: "Dropdown",
+                },
+            });
+
+            await testUtils.dom.click(dropdown.el.querySelector('button'));
+            assert.hasClass(dropdown.el, 'show');
```

Comment:
I think the `show` class is more of an implementation detail than anything, and using `isVisible` and `isNotVisible` would be more appropriate here.